docs/wcs/note_sip.rst

```diff
@@ -0,0 +1,40 @@
+.. include:: references.rst
+.. doctest-skip-all
+.. _note_sip:
+
+Note about SIP and WCS
+======================
+
+
+`astropy.wcs`, as well as `wcslib`_, include the `SIP`_ distortion
+in the WCS object if SIP coefficients are found in the header.
+The presence of a "-SIP" suffix in CTYPE is not used as a trigger
+to initialize the SIP distortion.
+
+It is important that headers implement correctly the SIP convention.
+If the intention is to use the SIP distortion, a header should have
+the SIP coefficients and the "-SIP" suffix in CTYPE.
+
+`astropy.wcs` prints INFO messages when inconsistent headers are detected,
+for example when SIP coefficients are present but CTYPE is missing a "-SIP" suffix.
+
+`astropy.wcs` will print a message about the inconsistent header
```

Comment:
[ignore the comment that I made about above paragraph and then deleted]

I would join this paragraph to the previous sentence